components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt

```diff
@@ -354,11 +354,11 @@ internal class GeckoPromptDelegate(private val geckoEngineSession: GeckoEngineSe
         val userName = geckoPrompt.authOptions.username ?: ""
         val password = geckoPrompt.authOptions.password ?: ""
         val method =
-            if (flags in GECKO_AUTH_FLAGS.HOST) AC_AUTH_METHOD.HOST else AC_AUTH_METHOD.PROXY
+            if (flags.contains(GECKO_AUTH_FLAGS.HOST)) AC_AUTH_METHOD.HOST else AC_AUTH_METHOD.PROXY
```

Comment:
Strange that the linter tells us to remove usages of `in` - that's a nicer extension operator imo. Which rule is it that does this?

Reply:
This was the `UnusedPrivateMember`. It doesn't recognize `in` is calling `Int.contains`

Reply:
Sounds like a bug maybe? @BraisGabin might be interested to know about this.

Reply:
Here was the `Int.contains()` https://searchfox.org/mozilla-mobile/source/android-components/components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/prompt/GeckoPromptDelegate.kt#678

Reply:
I saw this was fixed in 1.20 which is in RC so I just added this to the baseline.